1_fetch/src/fetch_data.R

```diff
@@ -1,15 +1,7 @@
-
-library(dplyr)
-library(readr)
-library(stringr)
-library(sbtools)
-library(whisker)
-
-
-
 # Get the data from ScienceBase
 fetch_data <- function() {
```

Comment:
One note about best practices for pipelining. For functions that are producing file output, we pass that filename through as a function argument. Here, even though `targets` knows it's a file target, and knows the name of that file because you are correctly returning the filename at the end of the function, a user can't tell what that filename is if they are looking at the `_targets.R`.  Also, if we pass the output filename as an argument, and the other file-specific parameters, like the science base ID and the name of the file being retrieved from science base, then we can conceivably use this exact same function to download a separate file. Try reworking this function and the target definition in `_targets.R` so that all of that information is passed in as arguments to the function.

Reply:
I did this, and am working on doing the same for the other target definitions.